app/tasks/first_task.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use src\Class\Cat;
+use src\Class\Dog;
+use src\Class\Bird;
+
+$cat = new Cat("Musya");
+$cat->eat();
```

Comment:
it doesn't show polymorphism. it's calling functions from different instances. try another way

Reply:
changed